base/access_provider/sync_to_target/model.go

```diff
@@ -41,11 +41,26 @@ type Access struct {
 }
 
 type WhoItem struct {
-	Users          []string `yaml:"users" json:"users"`
-	Groups         []string `yaml:"groups" json:"groups"`
-	InheritFrom    []string `yaml:"inheritFrom" json:"inheritFrom"`
-	UsersInGroups  []string `yaml:"usersInGroups" json:"usersInGroups"`
-	UsersInherited []string `yaml:"usersInherited" json:"usersInherited"`
+	// Users contains all account names of users assigned to this access provider
+	Users []string `yaml:"users,omitempty" json:"users,omitempty"`
```

Comment:
'directly assigned' may be more clear.

Also just say 'account names' instead of 'account names of users'.